src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -49,7 +49,7 @@ protected function setUp()
     }
 
     /**
-     * @expectedException LogicException
+     * @expectedException \LogicException
```

Comment:
yes, but phpstorm can't understand this and show that this class is undefined. This is only phpstorm problem? In other IDE it works well with "Find usage" functionality for example, or when you click on this class to check its source?